go.mod

```diff
@@ -0,0 +1,92 @@
+module github.com/openshift/source-to-image
+
+go 1.13
+
+replace (
+	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20170808145631-06b9cb351610
+	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
+	github.com/moby/buildkit => github.com/dmcgowan/buildkit v0.0.0-20170731200553-da2b9dc7dab9
+	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20190708153700-3bdd9d9f5532
+	google.golang.org/grpc => google.golang.org/grpc v1.3.0
```

Comment:
Why replace with v1.3.0?

Reply:
This is the tag corresponding to the commit-id used in `glide.lock`:

```
$ cd ${GOPATH}/src/google.golang.org/grpc
$ git name-rev --tags --name-only d2e1b51f33ff8c5e4a15560ff049d200e83726c5
v1.3.0
```